CIAOS-Rust/client/ciaos/config.py

```diff
@@ -0,0 +1,3 @@
+api_url= "http://127.0.0.1:8080" #modify your api url for server over here
```

Comment:
Create a Config object. We don't want users to go to site packages and modify this. 

We want users to create a config object just like in minio/boto check their config object. 

Something like this will be neat. 

```
import boto3
from botocore.config import Config

my_config = Config(
    region_name = 'us-west-2',
    signature_version = 'v4',
    retries = {
        'max_attempts': 10,
        'mode': 'standard'
    }
)

client = boto3.client('kinesis', config=my_config)

```

Reply:
so the parameters for config object in client 
would be api_url, userid, useraccesskey?

Reply:
Follow, Similar convention to minio/boto. For now we only require api_url and userid but in future if they pass all these we'll have it automatically without changes to the client